_data/projects/bitdust.yml

```diff
@@ -0,0 +1,22 @@
+name: BitDust
+desc: BitDust APP provides independent P2P on-line storage and secure private messaging
+site: https://bitdust.io
+tags:
+  - python
+  - git
```

Comment:
For most projects listed on the site we recommend 5 or so good labels - could this be trimmed down to the tags that are important to the project?

Reply:
sure! I put 5 tags. Thanks @shiftkey

Reply:
👍